src/restApiInstance.ts

```diff
@@ -35,12 +35,27 @@ const getNewRestApiInstanceAsync = async (
   requestId: RequestId,
   server: Server,
   jwtScopes: Set<JwtScopes>,
+  signal: AbortSignal,
   authInfo?: TableauAuthInfo,
 ): Promise<RestApi> => {
+  signal.onabort = () => {
+    log.info(
+      server,
+      {
+        type: 'request-cancelled',
+        requestId,
+        reason: signal.reason,
+      },
+      { logger: server.name, requestId },
+    );
+  };
```

Comment:
This isn't necessary, but it does send a signal back to the client that says "yep, we got your request to cancel any pending requests and we cancelled them!"